File: aula4/ex2/ex6.c
#include <stdio.h>

int main() {
    float num;
    printf("num: "); 
    scanf("%f", &num);

    if(num >0) {
        printf("Positivo"); 
    } else if(num==0) {
        printf("Neutro"); 
    } else {
        printf("Negativo");
    }
    scanf("%f", &num);
    return 0;
}

File: aula5/Ex6.java
import java.util.Scanner;

class Ex6 {
    public static void main(String[] args) {
        //Leia um número e exiba “Positivo” caso seja maior que zero.
        Scanner scanner = new Scanner(System.in);
        int num;
        System.out.println("Escreva seu número");
        num = scanner.nextInt();
        if(num > 0) {
            System.out.println("Número positivo");
        } else if(num == 0) {
            System.out.println("Valor neutro");
        } else {
            System.out.println("Número negativo");
        }
    }
}